### PR TITLE
Add boilerplate

### PR DESCRIPTION
**Pull request by chrisfoster76**

### CODE REVIEW

File: src/SFA.DAS.LevyTransferMatching.Infrastructure/SFA.DAS.LevyTransferMatching.Infrastructure.csproj [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (DannyGB) [inline src/SFA.DAS.LevyTransferMatching.Infrastructure/SFA.DAS.LevyTransferMatching.Infrastructure.csproj:4]: Could this be .netstandard as its a library?

File: src/SFA.DAS.LevyTransferMatching/Data/LevyTransferMatchingDbContext.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            base.OnModelCreating(modelBuilder);
+        }
+    }
+
+    public interface ILevyTransferMatchingDbContext
+    {
+        //DbSet<T> collection signatures go here
+    }
```
Comment (DannyGB) [inline src/SFA.DAS.LevyTransferMatching/Data/LevyTransferMatchingDbContext.cs:20]: I feel its better for interfaces to be in a separate file?

File: src/SFA.DAS.LevyTransferMatching/SFA.DAS.LevyTransferMatching.csproj [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (DannyGB) [inline src/SFA.DAS.LevyTransferMatching/SFA.DAS.LevyTransferMatching.csproj:4]: Could this be .netstandard?

File: src/SFA.DAS.LevyTransferMatching.Infrastructure/SFA.DAS.LevyTransferMatching.Infrastructure.csproj (hunk 1 above)
Comment (chrisfoster76) [reply src/SFA.DAS.LevyTransferMatching.Infrastructure/SFA.DAS.LevyTransferMatching.Infrastructure.csproj:4]: It _could_, but I didn't really see the need for it here?

File: src/SFA.DAS.LevyTransferMatching/Data/LevyTransferMatchingDbContext.cs (hunk 2 above)
Comment (chrisfoster76) [reply src/SFA.DAS.LevyTransferMatching/Data/LevyTransferMatchingDbContext.cs:20]: yes, will move it

**Review by DannyGB — APPROVED** [review]

**Pull request merged**